content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -65,7 +65,7 @@ The `.spec.schedule` field is required. The value of that field follows the [Cro
 # * * * * *
 ```
 
-For example, `0 0 13 * 5` states that the task must be started every Friday at midnight, as well as on the 13th of each month at midnight.
+For example, `0 0 13 * 5` means this cron job will run at midnight(12:00 AM) on Friday the 13th of every month.
```

Comment:
This is a weird crontab spec. The date has to be a Friday, and it has to be the 13th day of a month ... emm.
I'd suggest we change this to something more realistic.